packages/react-dom-bindings/src/client/ReactDOMInput.js

```diff
@@ -157,6 +178,20 @@ export function updateInput(
   if (checked != null && node.checked !== !!checked) {
     node.checked = checked;
   }
+
+  if (
+    name != null &&
+    typeof name !== 'function' &&
+    typeof name !== 'symbol' &&
+    typeof name !== 'boolean'
+  ) {
+    if (__DEV__) {
+      checkAttributeStringCoercion(name, 'name');
+    }
+    node.name = name;
+  } else {
+    node.removeAttribute('name');
```

Comment:
is it better to removeAttribute above instead of `= ''` so you don't need to do it here?